app/controllers/api_controller/chargeback.rb

```diff
@@ -0,0 +1,53 @@
+class ApiController
+  module Chargeback
+    # Request
+    # {
+    #   "action":"assign",
+    #
+    #   "data": [
+    #     {
+    #         "object_type": "tenant",
+    #         "object_id": 1
+    #     }
+    #   ]
+    # }
+    def chargebacks_assign_resource(object, _type, _id, data)
+      klass = Object.const_get(data['data'].first['object_type'].camelcase)
+      obj = klass.find(data['data'].first['object_id'])
+
+      ChargebackRate.set_assignments object.rate_type, [{:cb_rate => object, :object => obj}]
+
+      {:tags => object.tags}
+    end
+
+    # Request
+    # {
+    #   "action":"unassign",
+    #
+    #   "data": [
+    #     {
+    #         "object_type": "tenant",
+    #         "object_id": 1
+    #     }
+    #   ]
+    # }
+    def chargebacks_unassign_resource(object, _type, _id, data = nil)
+      obj_name = data['data'].first['object_type']
+      obj_id   = data['data'].first['object_id']
+      tag_name = "/chargeback_rate/assigned_to/#{obj_name}/id/#{obj_id}"
+
+      object.tags = object.tags.select { |t| t unless t.name == tag_name }
```

Comment:
reject! not implemented. I've done `object.tags = object.tags.reject { |t| t.name == tag_name }`